test/integration/api.bats

```diff
@@ -880,12 +945,43 @@ function teardown() {
 	[[ "${lines[1]}" != *"Paused"* ]]
 }
 
-# FIXME
 @test "docker version" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	# version
+	run docker_swarm version
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -ge 8 ]
+
+	# verify
+	client_reg='^Client version: [0-9]+\.[0-9]+\.[0-9]+$'
+	server_reg='^Server version: swarm/[0-9]+\.[0-9]+\.[0-9]+$'
+	[[ "${lines[0]}" =~ $client_reg ]]
+	[[ "${lines[5]}" =~ $server_reg ]]
 }
 
-# FIXME
 @test "docker wait" {
-	skip
+	TEMP_FILE=$(mktemp)
+	start_docker 3
+	swarm_manage
+
+	# run after 1 seconds, test_container will exit
+	run docker_swarm run -d --name test_container busybox sleep 1
+	[ "$status" -eq 0 ]
+
+	# make sure container exists and is up
+	run docker_swarm ps -l
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${lines[1]}" ==  *"test_container"* ]]
+	[[ "${lines[1]}" ==  *"Up"* ]]
+
+	# wait until exist(after 1 seconds)
+	timeout 5 docker -H $SWARM_HOST wait test_container > $TEMP_FILE
```

Comment:
I think you can `run timeout 5 ...` and just check `$output`, no need to redirect